src/borg/archiver.py

```diff
@@ -4715,6 +4715,8 @@ def define_borg_mount(parser):
                                help='number of weekly archives to keep')
         subparser.add_argument('-m', '--keep-monthly', dest='monthly', type=int, default=0,
                                help='number of monthly archives to keep')
+        subparser.add_argument('-q', '--keep-quarterly', dest='quarterly', type=int, default=0,
```

Comment:
Not sure if we want to "waste" "-q" for this, especially considering we'll need 2 different quarterly options and not both can have "-q".

Reply:
The long option name should be something making clear what one gets, considering there are different "quarters" according to WP.

Reply:
Should `--keep-13weekly` and `--keep-3monthly` be mutually-exclusive arguments? I don't think it makes sense to supply both args to one invocation, but I want a second opinion.

Reply:
Yes, mutually exclusive.